docker/types/services.py

```diff
@@ -306,9 +306,12 @@ class Resources(dict):
         mem_limit (int): Memory limit in Bytes.
         cpu_reservation (int): CPU reservation in units of 10^9 CPU shares.
         mem_reservation (int): Memory reservation in Bytes.
+        generic_reservations (dict): List of node level generic
```

Comment:
I understand you were looking to follow the same pattern as with `cpu_` and `mem_`, but I don't see a scenario where "generic limits" becomes a thing (especially because the Docker CLI already uses `--generic-resource` for this), so I think we should rename this to `generic_resources`

Reply:
yeah, for sure.